app/controllers/country_controller.rb

```diff
@@ -49,6 +49,7 @@ def load_vars
 
     @country or raise_404
 
+    @location = @country
```

Comment:
The idea of the shared variable name between countries and regions is to go in the partial as a `locals`. 
When you render a partial you can pass variables like so: `render partial: 'view_name', locals: {location: @country}`
That means you are rendering the partial and passing it a variable named location.
Then you can use that variable just like so: `location.method_name`

This means you don't need this line here or in the regions_controller

Reply:
Ah yes, of course! I will fix this now.